glean_parser/util.py

```diff
@@ -58,6 +58,22 @@ def ensure_list(value):
     return value
 
 
+def camelize(value):
```

Comment:
non blocking nit: these two functions only differ by the last argument passed to `inflection.camelize`. Do we really need 2 functions? If yes, can you mention their difference in their docstrings?

Reply:
These are used as filters in jinja2 templates, so for convenience and readability there, they have only one argument.  That second argument controls whether the first character in the output is upper case.